shipLHC/AdvTargetPoint.h

```diff
@@ -47,9 +50,14 @@ class AdvTargetPoint : public FairMCPoint
     int constexpr GetStrip() { return (fDetectorID) % 1024; }
     int constexpr GetModule() { return advsnd::target::columns * GetRow() + 1 + GetColumn(); }
     bool constexpr isVertical() { return GetPlane() == 0; };
+    TVector3 GetEntryPoint() const { return TVector3(2 * fX - exit_x, 2 * fY - exit_y, 2 * fZ - exit_z); }
```

Comment:
this is what I thought we want to avoid - lets just use the data from G4

Reply:
The calculation we wanted to avoid was using the momentum to extrapolate, which could cause effects of O(µm). Double precision should be orders of magnitude better than the detector resolution or approximations done when implementing the detector response.